ArmoryQt.py

```diff
@@ -2959,9 +2959,9 @@ def createCombinedLedger(self, resetMainLedger=False):
       if bdmState == BDM_BLOCKCHAIN_READY:
          for wltID in self.wltIDList:
             wlt = self.walletMap[wltID]
-            totalFunds += wlt.getBalance('Total')
-            spendFunds += wlt.getBalance('Spendable')
-            unconfFunds += wlt.getBalance('Unconfirmed')
+            totalFunds += int(wlt.getBalance('Total'))
```

Comment:
If I don't, I get the following errors on the CL. (This applies to the other change too.)

```
(ERROR) Traceback (most recent call last):
  File "/Users/droark/Projects/BitcoinArmory/osxbuild/workspace/Armory.app/Contents/MacOS/py/usr/lib/armory/ArmoryQt.py", line 5845, in handleCppNotification
    self.finishLoadBlockchainGUI()
  File "/Users/droark/Projects/BitcoinArmory/osxbuild/workspace/Armory.app/Contents/MacOS/py/usr/lib/armory/ArmoryQt.py", line 2882, in finishLoadBlockchainGUI
    self.createCombinedLedger()
  File "/Users/droark/Projects/BitcoinArmory/osxbuild/workspace/Armory.app/Contents/MacOS/py/usr/lib/armory/ArmoryQt.py", line 2962, in createCombinedLedger
    spendFunds += wlt.getBalance('Spendable')
TypeError: unsupported operand type(s) for +=: 'int' and 'SwigPyObject'

(ERROR) Traceback (most recent call last):
  File "/Users/droark/Projects/BitcoinArmory/osxbuild/workspace/Armory.app/Contents/MacOS/py/usr/lib/armory/armorymodels.py", line 81, in data
    dispStr = coin2str(bal, maxZeros=2)
  File "/Users/droark/Projects/BitcoinArmory/osxbuild/workspace/Armory.app/Contents/MacOS/py/usr/lib/armory/armoryengine/ArmoryUtils.py", line 1308, in coin2str
    nBtc = float(nSatoshi) / float(ONE_BTC)
TypeError: float() argument must be a string or a number
```